log_policy.md

```diff
@@ -18,18 +18,23 @@ This bug will be used to track all CT Logs operated by this Log Operator for as
 ### Existing CT Log Operators
 Once the Chrome team has confirmed the Log Operator’s contact information, or if an existing Log Operator is applying for additional CT Logs to be added to Chrome,  the CT Log Operator must next provide the following information about the new CT Logs in their existing CT Log Operator bug:
 * A description of the Logs, including applicable policies or requirements for logging certificates, and whether these Logs are compliant with RFC6962 or static-ct-api.
-* Public HTTP endpoints that respond to all Log Client Messages indicated in RFC 6962, Section 4, or Submission and Monitoring APIs specified in [c2sp.org/static-ct-api](https://c2sp.org/static-ct-api) v1.0, as appropriate.
-* The Logs’ public keys, attached as a binary file containing the DER encoding of the SubjectPublicKeyInfo ASN.1 structure.
-* The Maximum Merge Delay (MMD) of the Logs.
-* The set of Accepted Root Certificates of the Logs.
-* The expiry ranges of each Log in the form [rangeBegin, rangeEnd).
-	* Certificate expiry ranges for a set of Logs must be contiguous, with no gaps, and each expiry range should be between 6 and 12 months.
+* A JSON object (one per log) containing:
+    * a public HTTP endpoint that responds to all Log Client Messages indicated in RFC 6962, Section 4, or HTTP endpoints responding to Submission and Monitoring APIs specified in [c2sp.org/static-ct-api](https://c2sp.org/static-ct-api) v1.0, as appropriate.
+    * the Log's public key, provided as a DER-encoded ASN.1 SubjectPublicKeyInfo structure, base64-encoded,
+    * the SHA-256 hash of the Log's public key, base64-encoded (i.e. the LogID provided in SCTs issued by the log),
+    * the Maximum Merge Delay (MMD) of the Log, and
+    * the expiry range of the Log.
+* The initial set of Accepted Root Certificates of the Logs.
 * Whether the Logs will reject logging submissions for expired or revoked certificates.
```

Comment:
Does the decision to reject logging submissions for expired or revoked certificates often change for Log Operators? If not, is there value in adding this to the schema?